Dockerfile

```diff
@@ -88,11 +88,14 @@ RUN PATH="$PATH:/opt/mold/bin" \
     && PROFILE_DIR=$(if [ "$PROFILE" = dev ]; then echo debug; else echo $PROFILE; fi) \
     && mv target/$(xx-cargo --print-target-triple)/$PROFILE_DIR/qdrant /qdrant/qdrant
 
+# to skip the compilation for testing
+# RUN mkdir /qdrant/qdrant
+
 
 # Download and extract web UI
 RUN mkdir /static ; STATIC_DIR='/static' ./tools/sync-web-ui.sh
 
-FROM debian:12-slim AS qdrant
+FROM debian:11-slim AS qdrant
```

Comment:
is it just a merge artifact or intentional?

Reply:
Intentional because of https://github.com/devcontainers/features/issues/598. I have to see if there are other workarounds if this is a dealbreaker. This would be only temporary until the docker-in-docker feature is patched.

Reply:
We can ditch these changes now that we are relying on the `devcontainer.json` only